x-pack/test/accessibility/apps/users.ts

```diff
@@ -88,18 +88,21 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('a11y test for edit user panel', async () => {
+      await PageObjects.settings.clickLinkText('Users');
```

Comment:
NOTE TO REVIEWER
We need to add that because before we were using the breadcrumb to refresh the page but since we disable the link in the last breadcrumb. We need to click on the first breadcrumb to go back to the table and then click on the user.